docs/sourcecontrol/github.md

```diff
@@ -237,6 +237,6 @@ If you are using the [browser-based editor](/docs/remote/codespaces.md#browserba
 
 ![Continue Working On from web-based editor](images/github/codespaces-continue.png)
 
-The first time that you use **Continue Working On** with uncommitted changes, you will have the option to bring your edits to your selected development environment using **Edit Sessions**, which uses a VS Code service to store your pending changes.
+The first time that you use **Continue Working On** with uncommitted changes, you will have the option to bring your edits to your selected development environment using **Cloud Changes**, which uses a VS Code service to store your pending changes.
 
 These changes are deleted from our service once they are applied to your target development environment. If you choose to continue without your uncommitted changes, you can always change this preference later by configuring the setting `"workbench.editSessions.continueOn": "prompt"`.
```

Comment:
This setting also changed to `workbench.cloudChanges.continueOn`

Reply:
I'll fix in a follow up PR

Reply:
Here's the PR along with an unrelated broken link fix https://github.com/microsoft/vscode-docs/pull/5861